cli/targets/static.js

```diff
@@ -311,9 +311,15 @@ function buildFunction(type, functionName, gen, scope) {
         push("};");
 }
 
-function toJsType(field) {
+function toJsType(field, parentIsInterface) {
```

Comment:
Should we default this to `false`? This will also make the type clear.

```suggestion
function toJsType(field, parentIsInterface = false) {
```